react/Badge/Readme.md

```diff
@@ -28,3 +30,26 @@ import ButtonAction from 'cozy-ui/transpiled/react/ButtonAction';
   </p>
 </div>
 ```
+
+
+#### Alignment
+
+Only supports `bottom-right` 
+
+
+```
+import Badge from 'cozy-ui/transpiled/react/Badge'
+import Icon from 'cozy-ui/transpiled/react/Icon';
+
+
+
+<div>
+  <p>
+    <Badge 
+      content={ <Icon icon="link" size={10} />} type="normal" alignment="bottom-right" size='medium'>
+      <Icon icon="cloud" size="32" />
+    </Badge>
+  </p>
+  
+</div>
+```
```

Comment:
I am not sure why we add doc for `alignment` since it can only have one value ?

Reply:
For now, but we plan to add top-right pretty soon